modules/system/rolls/cast-test.js

```diff
@@ -44,6 +48,11 @@ export default class CastTest extends TestWFRP {
   runPreEffects() {
     super.runPreEffects();
     this.actor.runEffects("preRollCastTest", { test: this, cardOptions: this.context.cardOptions })
+    //@HOUSE
+    if (this.preData.unofficialGrimoire && this.preData.ingredientMode == 'power' && this.hasIngredient) {
+      this.preData.canReverse = true;
+    }
+    //@HOUSE
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("unofficialgrimoire")`

Reply:
done